modules/memfault-firmware-sdk/memfault_lte_metrics.c

```diff
@@ -221,7 +221,12 @@ void memfault_lte_metrics_update(void)
 
 	err = modem_info_get_operator(operator_name, sizeof(operator_name));
 	if (err != 0) {
-		LOG_WRN("Network operator collection failed, error: %d", err);
+		LOG_DBG("Network operator name was not reported by the modem");
+
+		err = MEMFAULT_METRIC_SET_STRING(ncs_lte_operator, "Not reported");
```

Comment:
"Not available" would probably be a little more "standard" in many context but feel free to choose what best fits into this context.

Reply:
That's true, changed it to "Not available". 
FYI @gminn